### ISSUE TITLE
Unexpected input(s) 'skip-go-installation', valid inputs are ['version', 'args', 'working-directory', 'github-token', 'only-new-issues']

### ISSUE THREAD

**Issue by Kmelow** [golangci/golangci-lint-action#155]
Hello,

I have a simplified mono-repo with a Github action, all bellow.

I took the example from the [How to use](https://github.com/golangci/golangci-lint-action#how-to-use) section and modified:
- `runs-on: ubuntu-20.04`
- `with.version: v1.36` (currently installed in my system)
- `skip-go-installation: true` because I want to control the Go version I'm using

Then I get the following warning:

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/46656358/106459290-ee8a2b00-6491-11eb-8ce9-90f5c2a005f2.png">

Any suggestions:

---

## Repo
```
.
└── modlinterabs
    ├── go.mod
    ├── greeterabs
    │   └── logic.go
    └── main.go
```
## Github Action Workflow
```yml
name: golangci-lint
on:
  push:
    tags:
      - v*
    branches:
      - master
      - main
  pull_request:
jobs:
  golangci:
    name: lint
    runs-on: ubuntu-20.04
    steps:
      - uses: actions/checkout@v2
      - name: Set up Go
        uses: actions/setup-go@v2
        with:
          go-version: 1.15
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v2
        with:
          version: v1.36
          working-directory: modlinterabs
          args: -v
          skip-go-installation: true
```

**Comment by pierrre** [comment]
I think it's not yet available in the latest tagged version.
Can you release a new version please ?

**Comment by SVilgelm** [comment]
it is available in latest v2.4.0 release, I also re-tagged the v2 to point for v2.4.0
So it should work
